 2019-03-15 08:19:33,126 [P3128/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.13.2 application starting on DESKTOP-4829FJV
 2019-03-15 08:19:33,247 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-03-15 08:19:33,338 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 90ms)
 2019-03-15 08:19:33,627 [P3128/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-03-15 08:19:33,633 [P3128/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-03-15 08:19:33,641 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:35,828 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2187ms)
 2019-03-15 08:19:35,829 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-03-15 08:19:35,834 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 4ms)
 2019-03-15 08:19:35,967 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:35,968 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:19:35,969 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-03-15 08:19:35,970 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 1ms)
 2019-03-15 08:19:35,976 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:35,977 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:19:35,978 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-03-15 08:19:35,983 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2019-03-15 08:19:36,043 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:36,044 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:19:36,044 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-03-15 08:19:36,046 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 1ms)
 2019-03-15 08:19:36,047 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:36,048 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:19:36,049 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-03-15 08:19:36,053 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 4ms)
 2019-03-15 08:19:41,713 [P3128/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-03-15 08:19:42,687 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-03-15 08:19:42,809 [P3128/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 121ms)
 2019-03-15 08:19:43,068 [P3128/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-03-15 08:19:43,638 [P3128/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-03-15 08:19:44,691 [P3128/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-03-15 08:19:44,740 [P3128/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 12093ms)
 2019-03-15 08:19:44,750 [P3128/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-03-15 08:19:44,755 [P3128/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-03-15 08:19:44,759 [P3128/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-03-15 08:19:49,499 [P3128/D2/T12] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:53679/umbraco
 2019-03-15 08:19:49,507 [P3128/D2/T12] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:53679/umbraco (UmbracoModule request)
 2019-03-15 08:19:51,049 [P3128/D2/T12] INFO  umbraco.content - Load Xml from file...
 2019-03-15 08:19:51,121 [P3128/D2/T12] INFO  umbraco.content - Loaded Xml from file.
 2019-03-15 08:19:51,941 [P3128/D2/T12] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2019-03-15 08:19:53,024 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:53,025 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:19:53,025 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-03-15 08:19:53,027 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 2ms)
 2019-03-15 08:19:55,130 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:55,130 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:19:55,131 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-03-15 08:19:55,133 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 1ms)
 2019-03-15 08:19:55,707 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:19:55,709 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:19:55,710 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.cms.businesslogic.macro.IMacroEngine
 2019-03-15 08:19:55,711 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.cms.businesslogic.macro.IMacroEngine (took 1ms)
 2019-03-15 08:20:10,334 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:20:10,334 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:20:10,335 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-03-15 08:20:10,337 [P3128/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 1ms)
 2019-03-15 08:20:10,927 [P3128/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:20:10,928 [P3128/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:20:10,930 [P3128/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-03-15 08:20:10,933 [P3128/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 3ms)
 2019-03-15 08:20:18,374 [P3128/D2/T11] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username vaiva.ignataviciute17@gmail.com from IP address ::1
 2019-03-15 08:20:18,381 [P3128/D2/T11] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: vaiva.ignataviciute17@gmail.com logged in from IP address ::1
 2019-03-15 08:20:19,932 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:20:19,933 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:20:19,934 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-03-15 08:20:19,937 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 2ms)
 2019-03-15 08:20:20,043 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:20:20,044 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:20:20,045 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-03-15 08:20:20,046 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 1ms)
 2019-03-15 08:20:20,081 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:20:20,082 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:20:20,083 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-03-15 08:20:20,084 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 1ms)
 2019-03-15 08:20:20,241 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:20:20,243 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:20:20,243 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-03-15 08:20:20,245 [P3128/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 1ms)
 2019-03-15 08:20:51,361 [P3128/D2/T10] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-03-14 08:20:51
 2019-03-15 08:21:06,055 [P3128/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:21:06,056 [P3128/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:21:06,057 [P3128/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-03-15 08:21:06,059 [P3128/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 1ms)
 2019-03-15 08:21:28,574 [P3128/D2/T19] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:21:28,575 [P3128/D2/T19] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 08:21:28,576 [P3128/D2/T19] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-03-15 08:21:28,577 [P3128/D2/T19] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2019-03-15 08:31:55,546 [P3128/D2/T64] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 08:31:55,547 [P3128/D2/T64] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 08:31:55,548 [P3128/D2/T64] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-03-15 08:31:55,626 [P3128/D2/T64] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 78ms)
 2019-03-15 08:32:15,671 [P3128/D2/T63] INFO  Umbraco.Core.Security.UmbracoMembershipProviderBase - Login attempt succeeded for username sam from IP address ::1
 2019-03-15 08:32:32,285 [P3128/D2/T69] INFO  Umbraco.Core.Security.UmbracoMembershipProviderBase - Login attempt succeeded for username emilijus from IP address ::1
 2019-03-15 08:51:23,394 [P3128/D2/T79] INFO  Umbraco.Core.Logging.OwinLogger - Event Id: 0, state: User logged will be logged out due to timeout: vaiva.ignataviciute17@gmail.com, IP Address: ::1
 2019-03-15 08:54:11,299 [P3128/D2/T81] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt failed for username emilijus from IP address ::1
 2019-03-15 08:54:16,580 [P3128/D2/T93] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username vaiva.ignataviciute17@gmail.com from IP address ::1
 2019-03-15 08:54:16,581 [P3128/D2/T93] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: vaiva.ignataviciute17@gmail.com logged in from IP address ::1
 2019-03-15 09:13:49,379 [P3128/D2/T157] INFO  Umbraco.Core.Logging.OwinLogger - Event Id: 0, state: User logged will be logged out due to timeout: vaiva.ignataviciute17@gmail.com, IP Address: ::1
 2019-03-15 09:44:35,696 [P9244/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.13.2 application starting on DESKTOP-4829FJV
 2019-03-15 09:44:35,803 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-03-15 09:44:35,872 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 69ms)
 2019-03-15 09:44:36,119 [P9244/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-03-15 09:44:36,124 [P9244/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-03-15 09:44:36,129 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 09:44:37,246 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1117ms)
 2019-03-15 09:44:37,247 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-03-15 09:44:37,279 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 31ms)
 2019-03-15 09:44:37,352 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 09:44:37,354 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2ms)
 2019-03-15 09:44:37,354 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-03-15 09:44:37,380 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 25ms)
 2019-03-15 09:44:37,382 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 09:44:37,383 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 09:44:37,384 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-03-15 09:44:37,416 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 32ms)
 2019-03-15 09:44:37,481 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 09:44:37,482 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 09:44:37,482 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-03-15 09:44:37,509 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 26ms)
 2019-03-15 09:44:37,512 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 09:44:37,513 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 09:44:37,514 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-03-15 09:44:37,539 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 24ms)
 2019-03-15 09:44:45,392 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6af0793c-2d1e-44f0-810e-f0972fa6b5ec
Error Number:948,State:1,Class:20
 2019-03-15 09:44:45,393 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:45,675 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-03-15 09:44:45,724 [P9244/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 49ms)
 2019-03-15 09:44:45,972 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6af0793c-2d1e-44f0-810e-f0972fa6b5ec
Error Number:948,State:1,Class:20
 2019-03-15 09:44:45,974 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:46,976 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6af0793c-2d1e-44f0-810e-f0972fa6b5ec
Error Number:948,State:1,Class:20
 2019-03-15 09:44:46,977 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:47,979 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6af0793c-2d1e-44f0-810e-f0972fa6b5ec
Error Number:948,State:1,Class:20
 2019-03-15 09:44:47,980 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:48,982 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6af0793c-2d1e-44f0-810e-f0972fa6b5ec
Error Number:948,State:1,Class:20
 2019-03-15 09:44:48,983 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:49,986 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:6af0793c-2d1e-44f0-810e-f0972fa6b5ec
Error Number:948,State:1,Class:20
 2019-03-15 09:44:49,987 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:51,053 [P9244/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: (LocalDB)\MSSQLLocalDB, InitialCatalog: . Exception: System.Data.SqlClient.SqlException (0x80131904): The database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF' cannot be opened because it is version 869. This server supports version 852 and earlier. A downgrade path is not supported.
Could not open new database 'C:\USERS\VAIVINIS\SOURCE\REPOS\AARHUSWEBDEVCOOP\AARHUSWEBDEVCOOP\APP_DATA\AARHUSWEBDEVCOOP.MDF'. CREATE DATABASE is aborted.
An attempt to attach an auto-named database for file C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\App_Data\AarhusWebDevCoop.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:0c80267c-417d-4281-95d2-d9b63994dae8
Error Number:948,State:1,Class:20
 2019-03-15 09:44:51,054 [P9244/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2019-03-15 09:44:51,062 [P9244/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   at Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   at Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   at Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2019-03-15 09:45:01,115 [P9244/D2/T6] INFO  Umbraco.Core.MainDom - Stopping...
 2019-03-15 09:45:01,117 [P9244/D2/T6] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-03-15 09:45:02,129 [P9244/D2/T6] INFO  Umbraco.Web.ExamineStartup - Complete (took 1012ms)
 2019-03-15 09:45:02,132 [P9244/D2/T6] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-03-15 09:45:32,177 [P9244/D2/T8] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: InitializationError

_shutDownMessage=Initialization Error
HostingEnvironment initiated shutdown
HostingEnvironment caused shutdown

_shutDownStack=   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   at System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)
   at System.Web.HttpRuntime.AppDomainShutdownTimerCallback(Object state)
   at System.Threading.TimerQueueTimer.CallCallbackInContext(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.TimerQueueTimer.CallCallback()
   at System.Threading.TimerQueueTimer.Fire()
   at System.Threading.TimerQueue.FireNextTimers()
   at System.Threading.TimerQueue.AppDomainTimerCallback()
 2019-03-15 10:58:56,701 [P9244/D3/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.13.2 application starting on DESKTOP-4829FJV
 2019-03-15 10:58:56,802 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-03-15 10:58:56,854 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 51ms)
 2019-03-15 10:58:57,110 [P9244/D3/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-03-15 10:58:57,117 [P9244/D3/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-03-15 10:58:57,125 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:58:58,530 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1404ms)
 2019-03-15 10:58:58,531 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-03-15 10:58:58,576 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 44ms)
 2019-03-15 10:58:58,655 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:58:58,656 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 10:58:58,657 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-03-15 10:58:58,692 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 35ms)
 2019-03-15 10:58:58,696 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:58:58,697 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 10:58:58,698 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-03-15 10:58:58,734 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 36ms)
 2019-03-15 10:58:58,811 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:58:58,812 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 10:58:58,813 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-03-15 10:58:58,843 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 30ms)
 2019-03-15 10:58:58,847 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:58:58,848 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 10:58:58,849 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-03-15 10:58:58,886 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 37ms)
 2019-03-15 10:59:04,805 [P9244/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-03-15 10:59:05,465 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-03-15 10:59:05,517 [P9244/D3/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 52ms)
 2019-03-15 10:59:05,787 [P9244/D3/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-03-15 10:59:06,409 [P9244/D3/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-03-15 10:59:06,412 [P9244/D3/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-03-15 10:59:06,602 [P9244/D3/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 10167ms)
 2019-03-15 10:59:06,620 [P9244/D3/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-03-15 10:59:06,626 [P9244/D3/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-03-15 10:59:06,632 [P9244/D3/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-03-15 10:59:07,422 [P9244/D3/T78] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:53679/umbraco
 2019-03-15 10:59:07,428 [P9244/D3/T78] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:53679/umbraco (UmbracoModule request)
 2019-03-15 10:59:08,147 [P9244/D3/T78] INFO  umbraco.content - Load Xml from file...
 2019-03-15 10:59:08,202 [P9244/D3/T78] INFO  umbraco.content - Loaded Xml from file.
 2019-03-15 10:59:10,321 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:59:10,322 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 10:59:10,323 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-03-15 10:59:10,353 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 30ms)
 2019-03-15 10:59:21,569 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:59:21,570 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 10:59:21,571 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-03-15 10:59:21,587 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 15ms)
 2019-03-15 10:59:22,164 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 10:59:22,165 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 10:59:22,167 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.cms.businesslogic.macro.IMacroEngine
 2019-03-15 10:59:22,189 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.cms.businesslogic.macro.IMacroEngine (took 22ms)
 2019-03-15 11:00:08,389 [P9244/D3/T79] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-03-14 11:00:08
 2019-03-15 11:00:33,879 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:33,880 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 11:00:33,882 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-03-15 11:00:33,923 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 41ms)
 2019-03-15 11:00:34,531 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:34,533 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:00:34,534 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-03-15 11:00:34,642 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 108ms)
 2019-03-15 11:00:41,943 [P9244/D3/T76] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username vaiva.ignataviciute17@gmail.com from IP address ::1
 2019-03-15 11:00:41,949 [P9244/D3/T76] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: vaiva.ignataviciute17@gmail.com logged in from IP address ::1
 2019-03-15 11:00:43,209 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:43,210 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:00:43,211 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-03-15 11:00:43,245 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 33ms)
 2019-03-15 11:00:43,314 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:43,315 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:00:43,316 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-03-15 11:00:43,316 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-03-15 11:00:43,332 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:43,333 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:00:43,334 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-03-15 11:00:43,385 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 50ms)
 2019-03-15 11:00:43,486 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:43,487 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:00:43,488 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-03-15 11:00:43,533 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 44ms)
 2019-03-15 11:00:59,184 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:00:59,185 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:00:59,186 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-03-15 11:00:59,333 [P9244/D3/T79] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 147ms)
 2019-03-15 11:04:07,379 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:04:07,380 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:04:07,381 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-03-15 11:04:07,428 [P9244/D3/T78] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 46ms)
 2019-03-15 11:14:17,666 [P9244/D3/T76] INFO  Umbraco.Core.Security.UmbracoMembershipProviderBase - Login attempt succeeded for username sam from IP address ::1
 2019-03-15 11:14:53,257 [P9244/D3/T76] INFO  Umbraco.Core.Security.UmbracoMembershipProviderBase - Login attempt succeeded for username emilijus from IP address ::1
 2019-03-15 11:15:50,594 [P9244/D3/T76] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:15:50,595 [P9244/D3/T76] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:15:50,595 [P9244/D3/T76] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2019-03-15 11:15:50,626 [P9244/D3/T76] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 31ms)
 2019-03-15 11:45:36,880 [P2304/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.13.2 application starting on DESKTOP-4829FJV
 2019-03-15 11:45:37,001 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-03-15 11:45:37,074 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 73ms)
 2019-03-15 11:45:37,398 [P2304/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-03-15 11:45:37,406 [P2304/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-03-15 11:45:37,416 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:45:39,153 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1737ms)
 2019-03-15 11:45:39,154 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-03-15 11:45:39,160 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2019-03-15 11:45:39,373 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:45:39,375 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2ms)
 2019-03-15 11:45:39,376 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-03-15 11:45:39,377 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 1ms)
 2019-03-15 11:45:39,380 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:45:39,381 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:45:39,382 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-03-15 11:45:39,388 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 6ms)
 2019-03-15 11:45:39,471 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:45:39,472 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:45:39,473 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-03-15 11:45:39,474 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 1ms)
 2019-03-15 11:45:39,478 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:45:39,479 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:45:39,480 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-03-15 11:45:39,484 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 4ms)
 2019-03-15 11:45:50,899 [P2304/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-03-15 11:45:51,965 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Models.PublishedContent.PublishedContentModel
 2019-03-15 11:45:52,102 [P2304/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Models.PublishedContent.PublishedContentModel (took 136ms)
 2019-03-15 11:45:52,442 [P2304/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-03-15 11:45:53,606 [P2304/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-03-15 11:45:53,609 [P2304/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-03-15 11:45:53,925 [P2304/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 17490ms)
 2019-03-15 11:45:53,942 [P2304/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalMemberIndexSet to be unlocked since it was left in a locked state
 2019-03-15 11:45:53,948 [P2304/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2019-03-15 11:45:53,969 [P2304/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2019-03-15 11:45:58,666 [P2304/D2/T7] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:53679/umbraco
 2019-03-15 11:45:58,676 [P2304/D2/T7] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:53679/umbraco (UmbracoModule request)
 2019-03-15 11:46:00,900 [P2304/D2/T7] INFO  umbraco.content - Load Xml from file...
 2019-03-15 11:46:00,910 [P2304/D2/T7] INFO  umbraco.content - Loaded Xml from file.
 2019-03-15 11:46:03,753 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:03,754 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:03,755 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-03-15 11:46:03,759 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 3ms)
 2019-03-15 11:46:07,312 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:07,313 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:07,314 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-03-15 11:46:07,317 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 2ms)
 2019-03-15 11:46:08,188 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:08,189 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:08,190 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.cms.businesslogic.macro.IMacroEngine
 2019-03-15 11:46:08,192 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.cms.businesslogic.macro.IMacroEngine (took 1ms)
 2019-03-15 11:46:17,791 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:17,792 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 11:46:17,793 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-03-15 11:46:17,794 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 1ms)
 2019-03-15 11:46:18,249 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:18,250 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:18,251 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2019-03-15 11:46:18,259 [P2304/D2/T7] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 7ms)
 2019-03-15 11:46:27,373 [P2304/D2/T11] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username vaiva.ignataviciute17@gmail.com from IP address ::1
 2019-03-15 11:46:27,378 [P2304/D2/T11] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: vaiva.ignataviciute17@gmail.com logged in from IP address ::1
 2019-03-15 11:46:28,412 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:28,413 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:28,414 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2019-03-15 11:46:28,416 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 2ms)
 2019-03-15 11:46:28,488 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:28,490 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:28,490 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2019-03-15 11:46:28,491 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2019-03-15 11:46:28,506 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:28,507 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-03-15 11:46:28,507 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2019-03-15 11:46:28,509 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 1ms)
 2019-03-15 11:46:28,575 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:28,577 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:28,578 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2019-03-15 11:46:28,579 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 1ms)
 2019-03-15 11:46:35,130 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 11:46:35,132 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-03-15 11:46:35,133 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.HealthCheck.HealthCheck
 2019-03-15 11:46:35,135 [P2304/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.HealthCheck.HealthCheck (took 2ms)
 2019-03-15 11:47:01,153 [P2304/D2/T11] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-03-14 11:47:01
 2019-03-15 12:10:59,394 [P2304/D2/T137] INFO  Umbraco.Core.Security.UmbracoMembershipProviderBase - Login attempt failed for username vaiva.ignataviciute17@gmail.com from IP address ::1, the user does not exist
 2019-03-15 12:10:59,405 [P2304/D2/T137] INFO  Umbraco.Web.Mvc.UmbracoPageResult - Executing Umbraco RouteDefinition controller
 2019-03-15 12:10:59,456 [P2304/D2/T137] INFO  Umbraco.Web.Mvc.UmbracoPageResult - Finished (took 50ms)
 2019-03-15 12:11:05,732 [P2304/D2/T137] INFO  Umbraco.Core.Security.UmbracoMembershipProviderBase - Login attempt succeeded for username emilijus from IP address ::1
 2019-03-15 12:12:53,208 [P2304/D2/T147] WARN  umbraco.macro - Error loading partial view macro (View: ~/Views/MacroPartials/NavigationMenu.cshtml). Exception: System.Web.HttpCompileException (0x80004005): C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\Views\MacroPartials\NavigationMenu.cshtml(16): error CS0103: The name 'root' does not exist in the current context
   at System.Web.Compilation.AssemblyBuilder.Compile()
   at System.Web.Compilation.BuildProvidersCompiler.PerformBuild()
   at System.Web.Compilation.BuildManager.CompileWebFile(VirtualPath virtualPath)
   at System.Web.Compilation.BuildManager.GetVPathBuildResultInternal(VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   at System.Web.Compilation.BuildManager.GetVPathBuildResultWithNoAssert(HttpContext context, VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   at System.Web.Compilation.BuildManager.GetVirtualPathObjectFactory(VirtualPath virtualPath, HttpContext context, Boolean allowCrossApp, Boolean throwIfNotFound)
   at System.Web.Compilation.BuildManager.GetCompiledType(VirtualPath virtualPath)
   at System.Web.Compilation.BuildManager.GetCompiledType(String virtualPath)
   at System.Web.Mvc.BuildManagerWrapper.System.Web.Mvc.IBuildManager.GetCompiledType(String virtualPath)
   at System.Web.Mvc.BuildManagerCompiledView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Core.Profiling.ProfilingView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Web.Mvc.ControllerExtensions.RenderViewResultAsString(ControllerBase controller, ViewResultBase viewResult)
   at Umbraco.Web.Macros.PartialViewMacroEngine.Execute(MacroModel macro, IPublishedContent content)
   at Umbraco.Web.Macros.PartialViewMacroEngine.Execute(MacroModel macro, INode node)
   at umbraco.macro.LoadPartialViewMacro(MacroModel macro)
   at umbraco.macro.renderMacro(Hashtable pageElements, Int32 pageId)
 2019-03-15 12:12:53,212 [P2304/D2/T147] WARN  umbraco.macro - Error loading Partial View (file: ~/Views/MacroPartials/NavigationMenu.cshtml). Exception: System.Web.HttpCompileException (0x80004005): C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\Views\MacroPartials\NavigationMenu.cshtml(16): error CS0103: The name 'root' does not exist in the current context
   at System.Web.Compilation.AssemblyBuilder.Compile()
   at System.Web.Compilation.BuildProvidersCompiler.PerformBuild()
   at System.Web.Compilation.BuildManager.CompileWebFile(VirtualPath virtualPath)
   at System.Web.Compilation.BuildManager.GetVPathBuildResultInternal(VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   at System.Web.Compilation.BuildManager.GetVPathBuildResultWithNoAssert(HttpContext context, VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   at System.Web.Compilation.BuildManager.GetVirtualPathObjectFactory(VirtualPath virtualPath, HttpContext context, Boolean allowCrossApp, Boolean throwIfNotFound)
   at System.Web.Compilation.BuildManager.GetCompiledType(VirtualPath virtualPath)
   at System.Web.Compilation.BuildManager.GetCompiledType(String virtualPath)
   at System.Web.Mvc.BuildManagerWrapper.System.Web.Mvc.IBuildManager.GetCompiledType(String virtualPath)
   at System.Web.Mvc.BuildManagerCompiledView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Core.Profiling.ProfilingView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Web.Mvc.ControllerExtensions.RenderViewResultAsString(ControllerBase controller, ViewResultBase viewResult)
   at Umbraco.Web.Macros.PartialViewMacroEngine.Execute(MacroModel macro, IPublishedContent content)
   at Umbraco.Web.Macros.PartialViewMacroEngine.Execute(MacroModel macro, INode node)
   at umbraco.macro.LoadPartialViewMacro(MacroModel macro)
   at umbraco.macro.renderMacro(Hashtable pageElements, Int32 pageId)
 2019-03-15 12:12:53,238 [P2304/D2/T147] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
System.Web.HttpCompileException (0x80004005): C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\Views\MacroPartials\NavigationMenu.cshtml(16): error CS0103: The name 'root' does not exist in the current context
   at System.Web.Compilation.AssemblyBuilder.Compile()
   at System.Web.Compilation.BuildProvidersCompiler.PerformBuild()
   at System.Web.Compilation.BuildManager.CompileWebFile(VirtualPath virtualPath)
   at System.Web.Compilation.BuildManager.GetVPathBuildResultInternal(VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   at System.Web.Compilation.BuildManager.GetVPathBuildResultWithNoAssert(HttpContext context, VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   at System.Web.Compilation.BuildManager.GetVirtualPathObjectFactory(VirtualPath virtualPath, HttpContext context, Boolean allowCrossApp, Boolean throwIfNotFound)
   at System.Web.Compilation.BuildManager.GetCompiledType(VirtualPath virtualPath)
   at System.Web.Compilation.BuildManager.GetCompiledType(String virtualPath)
   at System.Web.Mvc.BuildManagerWrapper.System.Web.Mvc.IBuildManager.GetCompiledType(String virtualPath)
   at System.Web.Mvc.BuildManagerCompiledView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Core.Profiling.ProfilingView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Web.Mvc.ControllerExtensions.RenderViewResultAsString(ControllerBase controller, ViewResultBase viewResult)
   at Umbraco.Web.Macros.PartialViewMacroEngine.Execute(MacroModel macro, IPublishedContent content)
   at Umbraco.Web.Macros.PartialViewMacroEngine.Execute(MacroModel macro, INode node)
   at umbraco.macro.LoadPartialViewMacro(MacroModel macro)
   at umbraco.macro.renderMacro(Hashtable pageElements, Int32 pageId)
   at Umbraco.Web.UmbracoComponentRenderer.RenderMacro(macro m, IDictionary`2 parameters, page umbracoPage)
   at Umbraco.Web.UmbracoComponentRenderer.RenderMacro(String alias, IDictionary`2 parameters, page umbracoPage)
   at Umbraco.Web.UmbracoComponentRenderer.RenderMacro(String alias, IDictionary`2 parameters)
   at Umbraco.Web.UmbracoComponentRenderer.RenderMacro(String alias, Object parameters)
   at Umbraco.Web.UmbracoHelper.RenderMacro(String alias)
   at ASP._Page_Views_Master_cshtml.Execute() in C:\Users\Vaivinis\source\repos\AarhusWebDevCoop\AarhusWebDevCoop\Views\Master.cshtml:line 38
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   at System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   at System.Web.WebPages.WebPageBase.<>c__DisplayClass3.<RenderPageCore>b__2(TextWriter writer)
   at System.Web.WebPages.HelperResult.WriteTo(TextWriter writer)
   at System.Web.WebPages.WebPageBase.Write(HelperResult result)
   at System.Web.WebPages.WebPageBase.RenderSurrounding(String partialViewName, Action`1 body)
   at System.Web.WebPages.WebPageBase.PopContext()
   at System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   at System.Web.Mvc.RazorView.RenderView(ViewContext viewContext, TextWriter writer, Object instance)
   at System.Web.Mvc.BuildManagerCompiledView.Render(ViewContext viewContext, TextWriter writer)
   at Umbraco.Core.Profiling.ProfilingView.Render(ViewContext viewContext, TextWriter writer)
   at System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResult(ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   at System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass21.<>c__DisplayClass2b.<BeginInvokeAction>b__1c()
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass21.<BeginInvokeAction>b__1e(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeAction(IAsyncResult asyncResult)
   at System.Web.Mvc.Controller.<BeginExecuteCore>b__1d(IAsyncResult asyncResult, ExecuteCoreState innerState)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncVoid`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   at System.Web.Mvc.Controller.EndExecuteCore(IAsyncResult asyncResult)
   at System.Web.Mvc.Controller.<BeginExecute>b__15(IAsyncResult asyncResult, Controller controller)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncVoid`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   at System.Web.Mvc.Controller.EndExecute(IAsyncResult asyncResult)
   at System.Web.Mvc.Controller.System.Web.Mvc.Async.IAsyncController.EndExecute(IAsyncResult asyncResult)
   at System.Web.Mvc.MvcHandler.<BeginProcessRequest>b__5(IAsyncResult asyncResult, ProcessRequestState innerState)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncVoid`1.CallEndDelegate(IAsyncResult asyncResult)
   at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   at System.Web.Mvc.MvcHandler.EndProcessRequest(IAsyncResult asyncResult)
   at System.Web.Mvc.MvcHandler.System.Web.IHttpAsyncHandler.EndProcessRequest(IAsyncResult result)
   at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   at System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)
 2019-03-15 12:15:35,729 [P2304/D2/T144] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-03-15 12:15:35,732 [P2304/D2/T144] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 4ms)
 2019-03-15 12:15:35,733 [P2304/D2/T144] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-03-15 12:15:35,738 [P2304/D2/T144] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 4ms)
